File: 58 Length of Last Word/58length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int i = s.size() - 1, res = 0;

        while (i > 0 && s[i] == ' '){i--;}
        while (i >= 0 && s[i] != ' '){
            i--;
            res++;
        }
        return res;
    }
};

File: 58 Length of Last Word/58length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        n = len(s) - 1
        res = 0
        while n > 0 and s[n] == " ":
            n -= 1

        while n >= 0 and s[n] != " ":
            n -= 1
            res += 1
        
        return res

